### PR TITLE
[Feat: #154168276] Add store model and schema

### PR DESCRIPTION
**Pull request by omorilewa**
#### What does this PR do?
- This PR adds the store model and the schema

#### Description of Task to be completed?
- add models folder that contains created models.
- add schema folder that contains all schemas.
- add relevent mutations and query for the model schema.
- add tests fro model and schema.

#### How should this be manually tested?
- Using GraphiQL, enter the URL of the graphql endpoint
- Test the mutations in the store schema

#### Any background context you want to provide?
- Initially, we had a `models.py` file and a `schema.py` file in the root directory. To avoid a clutter of models/schema, this PR attempts to seperate them into different files and place them in the appropriate directories.

#### What are the relevant pivotal tracker stories?
[154168276](https://www.pivotaltracker.com/story/show/154168276)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,10 @@
 
 - Run server locally with the `gunicorn config.wsgi` command.
 
+- Run test with `python3 manage.py test`
```
Comment (oreoluwade) [inline README.md]: 👍

File: food_basket/schema/user.py [hunk 2]
```diff
@@ -39,6 +38,7 @@ def mutate(self, cls, args, context, info):
         last_name = args.get('last_name')
         phone_number = args.get('phone_number')
 
+        print(User.objects)
```
Comment (oreoluwade) [inline food_basket/schema/user.py]: I think we used this while debugging.

File: schema.py [hunk 3]
```diff
@@ -3,12 +3,16 @@
 import food_basket.schema
 
 
-class Query(food_basket.schema.Query,  graphene.ObjectType):
+class Query(food_basket.schema.Query, graphene.ObjectType):
```
Comment (oreoluwade) [inline schema.py]: I think we should have a query here to get details about a store just as we have for the User. To this end, we'd probab;y have to have the query in the `store schema` then import it here.

File: food_basket/schema/user.py (hunk 2 above)
Comment (omorilewa) [reply food_basket/schema/user.py]: Thanks.

**Pull request merged**